projects/iucn/src/app/iucn-shared/component/multi-select/multi-select.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit, ChangeDetectionStrategy, Input, Output, EventEmitter, ChangeDetectorRef } from '@angular/core';
+import { SelectOption } from '../select/select.component';
+
+@Component({
+  selector: 'laji-multi-select',
+  templateUrl: './multi-select.component.html',
+  styleUrls: ['./multi-select.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class MultiSelectComponent implements OnInit {
+
+  @Input() value: string[] = [];
+  @Input() placeholder: string;
+  @Input() options: SelectOption[];
+
+  @Output() valueChange = new EventEmitter();
+
+  selected = '';
+
+  constructor(
+    private cd: ChangeDetectorRef
+  ) { }
+
+  ngOnInit(): void {
+  }
+
+  onSelect(value: string) {
+    const values = (this.value || []);
+    if (!values.includes(value)) {
+      this.value = [...values, value];
+      this.valueChange.emit(this.value);
+    }
+
+    this.selected = value;
+    setTimeout(() => {
+      this.selected = '';
+      this.cd.markForCheck();
+    }, 0);
```

Comment:
It clears the select element. It would look weird if the select displays the latest selected value when all the selected values are displayed below the select (you can see how it looks by testing the "Threat assessment zone" filter)